tee-worker/local-setup/launch.py

```diff
@@ -164,6 +167,28 @@ def setup_environment(offset, config, parachain_dir):
             for flag in config["workers"][0]["flags"]
         ]
 
+def setup_worker_log_level():
+    log_level_dic = {}
+    with open("./local-setup/worker-log-level-config.toml") as f:
+        log_data = toml.load(f)
+
+        # Section
+        for (section, item) in log_data.items():
+            log_level_string = "";
+            indx = 0
+
+            for (k, v) in item.items():
+                if indx == 0:
+                    log_level_string += v+","
```

Comment:
This means `RUST_LOG` should always be first ? Maybe the script could check value of the key instead of idx ?